notify/run.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+#
+# Notify
+#
+COMMIT_MESSAGE=$(echo "$COMMIT_MESSAGE" | head -n1 | sed 's/"/\\"/g')
+
+if [[ -n $DEFINED_JOB_STATUS ]]; then
+  JOB_STATUS=$DEFINED_JOB_STATUS
+fi
+
+case $JOB_STATUS in
+  success)
+    STATUS=":tada: Succeeded"
+    COLOR="good" # Green
+    ;;
+  failure)
+    STATUS=":cry: Failed"
+    COLOR="danger" # Red
+    ;;
+  running)
+    STATUS=":woman-running: Running"
+    COLOR="#ffbf00" # Amber
+    ;;
+  cancelled)
+    STATUS="Cancelled"
```

Comment:
idk what a good emoji for this would be

Reply:
I'm gonna go with `:heavy_multiplication_x`